src/util/simplify_expr.cpp

```diff
@@ -1826,6 +1788,44 @@ optionalt<std::string> simplify_exprt::expr2bits(
   return {};
 }
 
+const array_exprt *simplify_exprt::get_content(const refined_string_exprt &s)
```

Comment:
I do wonder whether this would better be done as a local `static` procedure (then taking an additional `namespacet` parameter?